support-frontend/assets/pages/[countryGroupId]/components/componentContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { from, space } from '@guardian/source/foundations';
+import type { ReactNode } from 'react';
+import { Container } from 'components/layout/container';
+
+const container = css`
+	> div {
+		padding: ${space[5]}px 10px;
+		${from.mobileLandscape} {
+			padding-left: ${space[5]}px;
+			padding-right: ${space[5]}px;
+		}
+		${from.desktop} {
+			max-width: 940px;
+		}
+	}
+`;
+
+export type ComponentContainerProps = {
+	children: ReactNode;
+	borderColor?: string;
+	sideBorders?: boolean;
+	topBorder?: boolean;
+	cssOverrides?: SerializedStyles;
+};
+
+export function ComponentContainer({
```

Comment:
Do we need to introduce `ComponentContainer` in this PR? It's not imported / used in any of the other modules?

Reply:
Yep, its an added complexity thats not named well. Have removed component `componentContainer.tsx` from here and references in the other 2 (poster/accordian) components.